managed_vms/analytics/main_test.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2016 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import re
+
+import pytest
+import responses
+
+
+@pytest.fixture
+def app(monkeypatch):
+    monkeypatch.setenv('GA_TRACKING_ID', '1234')
+
+    import main
```

Comment:
yar, follows the same pattern at `managed_vms/mailjet/main_test.py`. Basically you can't import main until the env vars are patched. This is actually a common pattern in test fixtures - hide how the actual object constructed from the tests.